src/app/forms/custom-sections/dimensions/dimensions.component.html

```diff
@@ -51,13 +51,13 @@
     suffix="mm"
   ></app-switchable-input>
 
-  <ng-container *ngIf="!isPsv">
+  <ng-container *ngIf="!isPsv && hasAxleSpacings">
```

Comment:
If you have a hgv and it was missing its axelSpacings for some reason, would this prevent the user from adding any? i.e. when editing, the control would not be rendered for them to enter data.

Reply:
It would, I cant imagine a world where you wouldn't. The only other piece of data I could imagine using is Axels then

Reply:
Is it expected that all trailers and HGVs have axelSpacings populated? If so you could leave it just as `*ngIf="!isPsv"`?

Reply:
Initially, the check was only `*ngIf="!isPsv"`. But this breaks some of the unit tests using mock data and some of our data in the DB is also crap, when using a VIN for an incomplete record from the DB, with the `hasAxleSpacings` check, this breaks the page.

So, I will leave this here for completeness' sake and we can remove it in the future when we start working with real data.

Reply:
We shouldn't be adding code to work around our dodgy test/mock data. I'm not familiar enough with the data, but if it is possible for a trailer or hgv to be added without any axelSpacings, then it's going to cause problems.
Will the same form be used to create new tech records?